elasticsearch/client/__init__.py

```diff
@@ -1387,4 +1540,3 @@ def field_caps(self, index=None, body=None, params=None):
         """
         return self.transport.perform_request('GET', _make_path(index,
             '_field_caps'), params=params, body=body)
-
```

Comment:
please configure your editor not to make changes like these. It makes the PR hard to review when they are unrelated stylistic changes in unrelated places.